src/tsweekly/tsweekly.entity.ts

```diff
@@ -44,10 +44,10 @@ export class TsWeekly extends BaseEntity {
   preview: string
 
   @ApiProperty()
-  @Column({ name: 'user_signed', nullable: true })
+  @Column({ type: 'date', name: 'user_signed', nullable: true })
```

Comment:
I think this one actually needs to be a timestamp, since it needs to include the time AND date

Reply:
The timestamp inlcudes date